lib/npg_warehouse/loader/pacbio/run.pm

```diff
@@ -174,6 +180,15 @@ sub _build_run_wells {
       my $tki = $self->_well_tracking_info($well->{'uniqueId'});
       my $run = $self->_run;
 
+      open my $id_product_script, q[-|],
+        join q[ ], 'python', $self->id_script,
+          $run->{'pac_bio_run_name'}, $well->{'well'}
+        or $self->logconfess ('Cannot generate id_product');
+      my $id_product = <$id_product_script>;
+      $id_product =~ s/\s//xms;
```

Comment:
Here we assume that we get a value. However, we actually have to check that. We also might want to check that we get a string of the expected length.